source/hardware/controllers/hercules_djcontrol_MIX.rst

```diff
@@ -0,0 +1,116 @@
+
+Decks
+~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`SYNC` button                                           Toggle :ref:`Sync Lock <master-sync>`.
+:hwlabel:`SHIFT` + :hwlabel:`SYNC` button                        Set deck as Sync leader.
+:hwlabel:`CUE` button                                            Specifies, plays or recalls temporary cue point.
+:hwlabel:`SHIFT` + :hwlabel:`CUE` button                         Return to the beginning of the track.
+Play button                                                      Play/Pause the current track.
+:hwlabel:`SHIFT` + Play button                                   Cue Stutter.
+:hwlabel:`VINYL` button                                          Toggle scratch mode (default: on)
+:hwlabel:`TEMPO` 												 Controls pitch/tempo ratio
+===============================================================  ==========================================
+
+Mixer
+~~~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`Crossfader` 				                             Fades between deck 1 and 2
+:hwlabel:`MASTER` 				                             	 Controls the master output volume
+:hwlabel:`HEADPHONE` knob				                         Controls the Headphone output volume
+:hwlabel:`Headphone` buttons				                     Toggles PFL/Cue (headphones) for specific deck
+:hwlabel:`SHIFT` + :hwlabel:`CUE MASTER`						 Toggles between headphone’s audio source (Cue / Master)
+:hwlabel:`SHIFT` + :hwlabel:`SPLIT ON/OFF` 						 Toggles split cue (headphones)
+:hwlabel:`Volume` 				                             	 Controls deck volume
+:hwlabel:`Filter/Bass` knob				                         Controls deck’s filter/ equalizer low frequencies
+:hwlabel:`Filter/Bass` button				                     Toggles knob control between Filter/Bass
+===============================================================  ==========================================
+
+
+Performance Pads
+~~~~~~~~~~~~~~~~
+
+Hot Cue Mode
+^^^^^^^^^^^^
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`Pad 1 - 4`                                             Set and trigger :term:`hotcue` 1 - 4
+:hwlabel:`SHIFT` + :hwlabel: `Pad 1 - 4`                         Delete :term:`hotcue` 1 - 4.
```

Comment:
According to some photos I found online, "Pad" is not actually part of the label.

```suggestion
Pad :hwlabel:`1` - :hwlabel:`4`                                  Set and trigger :term:`hotcue` 1 - 4
:hwlabel:`SHIFT` + Pad :hwlabel:`1` - :hwlabel:`4`               Delete :term:`hotcue` 1 - 4.
```

Reply:
The pads themselves are numbered, and this is more what I was referring to. Any suggestions for this ?

Reply:
Have a look at the suggested code. You could put just the numbers into the hwlabel.